src/views/SettingsPage.vue

```diff
@@ -159,6 +215,27 @@ export default {
             alert(error);
           });
       }
+    },
+    addNewUser() {
+      const vm = this;
+      var user = new AV.User();
+      user.setUsername(vm.pendingChanges.newEmailAddress);
+      user.setPassword(vm.pendingChanges.newUserPassword);
```

Comment:
You can chain operations together without repeating `user`, like:

`user.setUsername(...).setPassword(...).blahBlah().signUp().then(...)`